Source/Tools/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ add_subdirectory(CommonTools)
 if (NOT MINGW_BUILD)
   if (BUILD_FEXCONFIG)
     add_subdirectory(FEXConfig/)
+
+    find_package(Qt5 COMPONENTS Qml Quick Widgets)
+    if (Qt5_FOUND)
+      add_subdirectory(FEXQonfig/)
+    endif()
```

Comment:
Can we please use Qt 6 instead here? Qt 5 OSS is effectively EOL.

Reply:
Sadly Kubuntu 24.04 LTS ships Qt 5 by default and figuring out what Qt 6 packages are needed to compile (and to not make it use the fallback theme!) seems to be nontrivial.

Supporting both simultaneously (and preferring Qt 6 over 5) should hopefully be possible without too much extra work though. The main breaking changes are in the Dialog APIs, which we should be able to handle via abstraction however.

Reply:
I created a compatibility module to deal with the dialogs (surprisingly tricky to make Qt conditionally import the right one). Both Qt 5 and Qt 6 are supported now!

My testing with Qt 6 was limited due to what Ubuntu currently packages, so feel free to give it a spin if you want :)